pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "pythonmonkey"
+version = "0.1.0"
+description = ""
+authors = ["Caleb Aikens <caleb@distributive.network>", "Tom Tang <xmader@distributive.network>"]
+readme = "README.md"
+packages = [
+  { include = "pythonmonkey", from = "python" },
+]
+include = [ "python/pythonmonkey/pythonmonkey.so" ]
+
+
+[tool.poetry.dependencies]
+python = "^3.9"
+
+
+[tool.poetry.build]
+script = "build.py"
+generate-setup-file = false 
+
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.3.1"
+pip = "^23.1.2"
```

Comment:
`numpy` is also [required for testing purposes](https://github.com/Distributive-Network/PythonMonkey/blob/b4e73ad/tests/python/test_pythonmonkey_eval.py#L934-L943).

Reply:
Got it, will add

Reply:
Done